src/main/java/core/service/mapper/impl/MoviesSessionMapperImpl.java

```diff
@@ -0,0 +1,46 @@
+package core.service.mapper.impl;
+
+import core.model.CinemaHall;
+import core.model.Movie;
+import core.model.MovieSession;
+import core.model.dto.MovieSessionRequestDto;
+import core.model.dto.MovieSessionResponseDto;
+import core.service.CinemaHallService;
+import core.service.MovieService;
+import core.service.mapper.MovieSessionMapper;
+import java.time.LocalDateTime;
+import org.springframework.stereotype.Component;
+
+@Component
+public class MoviesSessionMapperImpl implements MovieSessionMapper {
+    private final MovieService movieService;
+    private final CinemaHallService cinemaHallService;
+    
+    public MoviesSessionMapperImpl(MovieService movieService,
+                                   CinemaHallService cinemaHallService) {
+        this.movieService = movieService;
+        this.cinemaHallService = cinemaHallService;
+    }
+    
+    @Override
+    public MovieSessionResponseDto mapToDto(MovieSession movieSession) {
+        MovieSessionResponseDto responseDto = new MovieSessionResponseDto();
+        responseDto.setId(movieSession.getId());
+        responseDto.setMovieTitle(movieSession.getMovie().getTitle());
+        responseDto.setCinemaHallCapacity(movieSession.getCinemaHall().getCapacity());
+        responseDto.setCinemaHallDescription(movieSession.getCinemaHall().getDescription());
+        responseDto.setShowTime(movieSession.getShowTime());
```

Comment:
```suggestion
        responseDto.setShowTime(movieSession.getShowTime().toString());
```

Reply:
Thanks for the review. The setter thing is super stupid from my side. Somehow I missed it.